src/cli/format.js

```diff
@@ -437,11 +437,11 @@ async function formatFiles(context) {
       } else if (!context.argv.check && !context.argv.listDifferent) {
         const message = `${chalk.grey(fileNameToDisplay)} ${
           Date.now() - start
-        }ms`;
+        }ms (unchanged)`;
         if (isCacheExists) {
-          context.logger.log(`${message} (cached)`);
+          context.logger.debug(`${message} (cached)`);
```

Comment:
I prefer `context.logger.log`.

@fisker What do you think?

Reply:
Same here.